lib/lotus.rb

```diff
@@ -9,4 +10,18 @@
 #
 # @see http://lotusrb.org
 module Lotus
+
+  # @since x.x.x
+  #
```

Comment:
What do you think about add description for this method, something like `Return the current environment`